### PR TITLE
added:-  getFilterValue function will provide the input value of filt…

### PR DESCRIPTION
**Pull request by sdei-lovepreet**
getFilterValue function is added which will provide the input value of filter to the picky dropdown, so that if we have a larger list of options then we can only supply the matching options based on this value

### CODE REVIEW

File: src/Picky.js [hunk 1]
```diff
@@ -474,6 +481,7 @@ Picky.defaultProps = {
   filterDebounce: 150,
   dropdownHeight: 300,
   onChange: () => {},
+  getFilterValue: (value) => console.log('value---', value),
```
Comment (Aidurber) [inline src/Picky.js]: @sdei-lovepreet Don't give it a default value, that will break it for everyone else. Remove this line and your `if (this.props.getFilterValue)` will work for everyone.

**Comment by Aidurber** [comment]
@sdei-lovepreet Also, please add it to the README as documentation.

File: src/Picky.js (hunk 1 above)
Comment (sdei-lovepreet) [reply src/Picky.js]: ok i will

**Comment by Aidurber** [comment]
@sdei-lovepreet Looks good to me! Thank you for this PR.

**Pull request merged**